java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S1607_java.html

```diff
@@ -1,7 +1,8 @@
 <p>When a test fails due, for example, to infrastructure issues, you might want to ignore it temporarily. But without some kind of notation about why
 the test is being ignored, it may never be reactivated. Such tests are difficult to address without comprehensive knowledge of the project, and end up
 polluting their projects.</p>
-<p>This rule raises an issue for each ignored test that does not have a notation about why it is being skipped.</p>
+<p>This rule raises an issue for each ignored test that does not have a <code>WorkItem</code> attribute nor a comment about why it is being skipped on
```

Comment:
This is wrong. Apparently some changes has been made on initial RSPEC (code is in C#), but we lost change in Java.
I updated JAVA-specific RSPEC to restore previous description.